vnext/RNW.props

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
```

Comment:
“RNW” is rather unclear.
This could use a more MSBuild-friendly name, say, `React.Common.props`

Reply:
move it bundle.props